swift/ITSClient/Sources/ITSCore/Bootstrap/Data/ITSBootstrapResponseDTO.swift

```diff
@@ -0,0 +1,81 @@
+/*
+ * Software Name : ITSClient
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
+ *
+ * Software description: Swift ITS client.
+ */
+
+import Foundation
+
+struct ITSBootstrapResponseDTO: Decodable {
+    let identifier: String
+    let user: String
+    let password: String
+    let services: ITSBootstrapServiceDTO
+
+    enum CodingKeys: String, CodingKey {
+        case identifier = "iot3_id"
+        case user = "psk_iot3_id"
+        case password = "psk_iot3_secret"
+        case services
+    }
+}
+
+struct DynamicKey: CodingKey {
+    var stringValue: String
+    var intValue: Int?
+
+    init?(stringValue: String) {
+        self.stringValue = stringValue
+    }
+
+    init?(intValue: Int) {
+        nil
```

Comment:
Why `nil`? The `intValue`property will be never asigned

Reply:
Oups, I forgot to delete it. It's not used anymore. But it's for the CodingKey conformance. I will remove it.